client/src/views/home/HomePageHeader.vue

```diff
@@ -157,6 +157,84 @@ defineEmits<{
   }
 }
 
+.topbar-right {
```

Comment:
Redundant changes, there are three instances of `.topbar-right`, the second & third should be removed